suites/squid/rbd/tier-2_rbd_regression.yaml

```diff
@@ -238,3 +238,20 @@ tests:
       module: test_rbd_python_module.py
       polarion-id: CEPH-83574791
       name: Test image creation, write and read data using python rbd module
+
+  - test:
```

Comment:
please add this test in this suite as it's related to the group feature
https://github.com/red-hat-storage/cephci/blob/master/suites/squid/rbd/tier-2_rbd_group.yaml

Reply:
Addressed